sale_order_product_recommendation/wizards/sale_order_recommendation.py

```diff
@@ -51,16 +50,13 @@ def _default_order_id(self):
 
     def _recomendable_sale_order_lines_domain(self):
         """Domain to find recent SO lines."""
-        start = datetime.now() - timedelta(days=self.months * 30)
-        start = fields.Datetime.to_string(start)
         other_sales = self.env["sale.order"].search(
             [
                 (
                     "partner_id",
                     "child_of",
                     self.order_id.partner_id.commercial_partner_id.id,
                 ),
-                ("date_order", ">=", start),
```

Comment:
You mustn't remove this condition, as it's part of the features of the module.

Reply:
This condition has been reinstated.